### DISCUSSION TITLE
Emonesp

### DISCUSSION THREAD

**Discussion by ApiFelix** [FredM67/PVRouter-3-phase#87] [General]
Bonsoir
J'ai vu passer dans les release le support d emonesp
On peut donc simplement envoyer en wifi les donnees lues par les capteur?
Merci

**Comment by FredM67** [comment]
Oui, on peut envoyer les données, par exemple température(s), via un emonesp (c'est un esp32 ou équivalent avec le prog emonesp modifié qui tourne dessus).
J'ai déjà testé "sur mon bureau", cela fonctionne...

**Reply by ApiFelix** [reply]
Merci
Comment tu parles entre l'esp32 et le routeur?
Tu prends où l'alimentation de l'esp32?
Merci

**Reply by FredM67** [reply]
Les 2 communiquent par interface série.
Pour l'alim, tout dépend du module ESP32, selon que c'est un "brut", ou pas.
Dans tous les cas, les sorties ESP32 sont en 3.3V, donc si le mk2 est en 5V, il faut un diviseur de tension (RX/TX de l'esp32 sont potentiellement tolérants au 5V, mais bon, je ne sais plus si c'est officiel ou pas).

**Reply by ApiFelix** [reply]
😁👍
